jenkins/combirepo_run.jenkinsfile

```diff
@@ -329,6 +329,7 @@ pipeline {
     }
     success {
         echo 'Successfully!'
+        archiveArtifacts artifacts: 'mic-output/*'
```

Comment:
Aren't there any sub-directories?

Reply:
Yes, there should be no subdirectories.
Do we need to store logs, kickstart file, mic.conf or something else too?

Reply:
I'd prefer to save all meta-information we have during build, including repository information (kind of packages.list with hashes we have on firmware download  servers)